src/modules/testimonials/testimonials.service.ts

```diff
@@ -0,0 +1,42 @@
+import { HttpStatus, Injectable, NotFoundException } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { User } from '../user/entities/user.entity';
+import { CreateTestimonialDto } from './dto/create-testimonial.dto';
+import { Testimonial } from './entities/testimonials.entity';
+
+@Injectable()
+export class TestimonialsService {
+  constructor(
+    @InjectRepository(Testimonial)
+    private readonly testimonialRepository: Repository<Testimonial>,
+
+    @InjectRepository(User)
+    private readonly userRepository: Repository<User>
+  ) {}
+  async create(createTestimonialDto: CreateTestimonialDto, userId: string) {
+    const { content, name } = createTestimonialDto;
+
+    const user = await this.userRepository.findOneBy({ id: userId });
```

Comment:
there is a method in the user service that takes care of this, kindly reference the method and don't re-invent the wheel. 

UserService.getUserRecord

Reply:
Done